docs/source/intro/installation.rst

```diff
@@ -361,21 +361,24 @@ Using virtual environments
 ==========================
 
 AiiDA depends on a number of third party python packages, and usually on specific versions of those packages.
-In order not to interfere with third party packages needed by other software on your system, we **strongly** recommend isolating AiiDA in a virtual python environment.
+In order to not interfere with third party packages needed by other software on your system, we **strongly** recommend isolating AiiDA in a virtual python environment, such as the methods described below.
 
 .. admonition:: Additional Information
    :class: seealso title-icon-read-more
 
    A very good tutorial on Python environments is provided by `realpython.com <https://realpython.com/effective-python-environment>`__.
 
-`venv <https://docs.python.org/3/library/venv.html>`__ is module included directly with python for creating virtual environments.
+venv
+----
+
+`venv <https://docs.python.org/3/library/venv.html>`__ is a module included directly with python for creating virtual environments.
 To create a virtual environment, in a given directory, run:
 
 .. code-block:: console
 
    $ python3 -m venv /path/to/new/virtual/environment/aiida
 
-The command to activate the environment is shell specific (see `the documentation <https://docs.python.org/3/library/venv.html#creating-virtual-environments>`__.
+The command to activate the environment is shell specific (see `the documentation <https://docs.python.org/3/library/venv.html#creating-virtual-environments>`__).
```

Comment:
Do we need the literal dollar signs in front of console commands? Were they not inserted automatically? At least those that are not prefaced by the `(aiida)` virtual env indicator I imagine

Reply:
>  Were they not inserted automatically? 

Nope, to the best of my knowledge, there's no automatic insertion.